bedrock/firefox/templates/firefox/new/privacy-dmt/scene2.html

```diff
@@ -0,0 +1,98 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/scene2.html" %}
+
+{% block experiments %}{% endblock %}
+
+{% block content %}
+<main class="main mzp-t-firefox mzp-t-dark">
+  <section class="status">
+    <div class="mzp-l-content mzp-t-firefox windows-xpvista">
+      <p>{{ _('You’re using an insecure, outdated operating system <a href="%(url)s">no longer supported by Firefox</a>.')|format(url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista') }}</p>
+    </div>
+    <div class="mzp-l-content mzp-t-firefox">
+      {# fallback_url is replaced by the platform download link via JS, but if
+        something fails the user should still get a link to a working download path. #}
+
+      {% set id, fallback_url = 'direct-download-link', url('firefox.all') %}
+      <p>
+        {% if l10n_has_tag('firefox_new_try_again_link_020818') %}
+          {% trans %}
+            Your download should begin automatically. Didn’t work? <a id="{{ id }}" href="{{ fallback_url }}">Try downloading again</a>.
+          {% endtrans %}
+        {% else %}
+          {% trans %}
+            Your download should begin automatically. If it doesn’t, <a id="{{ id }}" href="{{ fallback_url }}">click here</a>.
+          {% endtrans %}
+        {% endif %}
+      </p>
+    </div>
+  </section>
+
+  <div class="mzp-l-content">
+    <div class="header-logos">
+      <h2><a class="firefox" href="{{ url('firefox') }}" title="{{ firefox_logo_title }}">{{ high_res_img('logos/firefox/logo-quantum-wordmark-white.png', {'alt': firefox_logo_title, 'width': '133', 'height': '50'}) }}</a></h2>
+      <h2><a class="mozilla" href="{{ url('mozorg.home') }}" title="{{ mozilla_logo_title }}"><img src="{{ static('img/logos/mozilla/wordmark-dark.svg')}}" alt="{{ mozilla_logo_title }}" width="101" height="34"></a></h2>
+    </div>
+
+    <div class="content-main">
+      <div class="download-button-wrapper desktop" id="download-button-wrapper-desktop">
+        {{ download_firefox(force_direct=true, dom_id='primary-download-button') }}
+      </div>
+
+
+      <h1 class="headline">
+      {% block headline %}
+        {{ _('Thank you for downloading Firefox. Updates coming soon.') }}
```

Comment:
Not sure what we can do to change this, but I could find this headline hugely confusing as a user? 😕

(have I downloaded the latest Firefox or not?)